ajo/in-app-messages/public/script.js

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+function createIdentityPayload(
+  id,
+  authenticatedState = "ambiguous",
+  primary = true
+) {
+  if (id.length === 0) {
+    return undefined;
+  }
+
+  return {
+    id,
+    authenticatedState,
+    primary,
+  };
+}
+
+function sendDisplayEvent(proposition) {
+  const { id, scope, scopeDetails = {} } = proposition;
+
+  alloy("sendEvent", {
+    xdm: {
+      eventType: "decisioning.propositionDisplay",
+      _experience: {
+        decisioning: {
+          propositions: [
+            {
+              id: id,
+              scope: scope,
+              scopeDetails: scopeDetails,
+            },
+          ],
+          propositionEventType: {
+            display: 1
+          },
+        },
+      },
+    },
+  });
+}
+
+function sendInteractEvent(label, proposition) {
```

Comment:
remove code from this file (and others) if it is not needed for this specific sample.  That avoids confusion.

Reply:
done